int32 x
int32 y
int32 lane_id  # Red == lane_id: 0, Green == lane_id: 1, Blue == lane_id: 2, Yellow == lane_id: 3
